sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -477,7 +484,7 @@ public virtual Response UpdateComponent(string digitalTwinId, string componentPa
         /// }
         /// </code>
         /// </example>
-        public virtual AsyncPageable<string> GetRelationshipsAsync(string digitalTwinId, string relationshipName = null, CancellationToken cancellationToken = default)
+        public virtual AsyncPageable<string> GetRelationshipsAsync(string digitalTwinId, string relationshipName = null, DigitalTwinsListRelationshipsOptions options = null, CancellationToken cancellationToken = default)
```

Comment:
Consider renaming this type from `DigitalTwinsListRelationshipsOptions` to `DigitalTwinsGetRelationshipsOptions` to stay consistent with the API naming.